ArrtModel/Model/ModelEditor/MaterialColor.cpp

```diff
@@ -27,32 +27,19 @@ MaterialColor::MaterialColor(ArrSessionManager* sessionManager, QObject* parent)
 
 RR::ApiHandle<RR::ColorMaterial> MaterialColor::getMaterial()
 {
-    if (m_material)
+    if (m_material && m_material->GetValid() && m_material->GetMaterialSubType() == RR::MaterialType::Color)
     {
```

Comment:
In general:
if (m_material && m_material->GetValid()
is the same as just
if (m_material->GetValid()

Reply:
TBH I wouldn't mind having a method in the ApiHandle that does both checks, since I don't think I need that distinction. The only (strong?) implication is that T has to implement GetValid